phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.css

```diff
@@ -218,11 +218,15 @@
 /*******************************************************/
 /************************ logo *************************/
 /*******************************************************/
-#logoText {
+#logo {
+    -fx-background-image: url('/org/phoenicis/javafx/views/common/phoenicis.png');
+    -fx-background-size: 1.8em;
+    -fx-background-repeat: no-repeat;
+    -fx-background-position: left center;
+    -fx-label-padding: 0 0 0 2.2em;
+    -fx-vpos: center;
     -fx-font-family: "Maven Pro";
     -fx-font-size: 1.8em;
```

Comment:
That's not correct. The font size for the logo is defined relative to the root font size. (Nevertheless I've tried it. Didn't work.)